Malay-Kumar-Parida/15.Kubernetes/04.ConfigMap_Secrets/example_config2.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: person2
+data:
+  author: "My_Enemy"
```

```suggestion
  author: "My_Enemy"
```
what is this enemy? use better naming